autoapi/mappers/base.py

```diff
@@ -47,8 +49,9 @@ class PythonMapperBase(object):
     # Create a page in the output for this object.
     top_level_object = False
 
-    def __init__(self, obj, options=None, jinja_env=None, url_root=None):
+    def __init__(self, obj, path, options=None, jinja_env=None, url_root=None):
```

Comment:
Any reason this was added on the base python mapper class? This breaks all of our implementations. Seems it could just live on the Python mapper.